BUILD

```diff
@@ -67,3 +68,17 @@ toolchain(
     toolchain = ":py_runtime_pair",
     toolchain_type = "@rules_python//python:toolchain_type",
 )
+
+# TODO
+write_file(
+    name = "ov_plugins_xml",
+    out = "_solib_linux_x86_64/plugins.xml",
```

Comment:
I don't like creating this file out of the root-directory BUILD, but I didn't see another way to get it in the right place. I'm also concerned about the use of the presumably machine-specific `_solib_linux_x86_64`, but I'm going to look for the output of the CI machines to see what happens with that.